src/include/daos_prop.h

```diff
@@ -67,7 +67,24 @@ enum daos_pool_props {
 	 * The pool svc rank list.
 	 */
 	DAOS_PROP_PO_SVC_LIST,
+	/**
+	 * Pool cell size.
+	 */
 	DAOS_PROP_PO_EC_CELL_SZ,
+	/**
+	 * Pool redundancy factor.
+	 */
+	DAOS_PROP_PO_REDUN_FAC,
+	/**
+	 * The pool performance domain affinity level
+	 * of EC object.
+	 */
+	DAOS_PROP_PO_EC_PDA,
+	/**
+	 * The pool performance domain affinity level
+	 * of replicated object.
+	 */
+	DAOS_PROP_PO_RP_PDA,
```

Comment:
should these be at the end?  You are changing the value of DAOS_PROP_PO_POLICY.  Otherwise, this is actually an ABI breaking change and a major version update, no?

Reply:
This is a mischanged by rebasing, let me fix it.